src/Heading.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const Heading = () => {
+  return (
+    <div className="header-container">
+      <img
+        className="logo-img"
+        src="https://image.flaticon.com/icons/svg/139/139899.svg"
+      />
+      <header className="App-header">CYF Hotel</header>
+    </div>
+  );
+};
+
+export default Heading;
```

Comment:
Good job. Do you think we could receive this `CYF Hotel` via props?

Reply:
Yes, we could definitely.